mobile/package.json

```diff
@@ -14,11 +14,11 @@
     "react-native": "^0.59.1",
     "react-native-camera": "^2.0.1",
     "react-native-video": "^4.4.0",
-    "react-native-youtube": "1.1.0",
+    "react-native-youtube": "^1.1.0",
```

Comment:
This one should stay a fixed version, since a specific version will be on the mobile apps.